cli/provider/cmd.go

```diff
@@ -173,7 +173,18 @@ func (c *CmdConfigurator) AddFlags(cmd *cobra.Command) error {
 	//add flags
 	var err error
 	cmd.Flags().SetNormalizeFunc(aliasNormalizeFunc)
+	cmd.Flags().String("configPath", ".", "Path to the local directory where keploy configuration file is stored")
+
 	switch cmd.Name() {
+
+	case "generate", "download":
+		cmd.Flags().StringSliceP("services", "s", c.cfg.Contract.Services, "Specify the services for which to generate/download contracts")
+		cmd.Flags().StringSliceP("tests", "t", c.cfg.Contract.Tests, "Specify the tests for which to generate/download contracts")
+		cmd.Flags().StringP("path", "p", c.cfg.Contract.Path, "Specify the path to generate/download contracts")
```

Comment:
rather than passing c.cfg.Contract.Path, you can just write "." so that it gets the default path as the current directory.